.github/workflows/check_tpch_queries.yml

```diff
@@ -0,0 +1,28 @@
+name: Tests for TPCH Queries
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  validate-queries:
+    if: ${{ github.event.label.name == 'full-test' }}
+    strategy:
+      matrix:
+        python-version: ["3.12"]
+        os: [ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install uv 
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
+      - name: install-reqs
+        run: uv pip install --upgrade -r tpch/requirements-tpch.txt --system
```

Comment:
instead of this, would we first do `pip install -e .` so that we install Narwhals locally and use that? then, if a PR introduces anything which affects the tpc-h queries, we'll know right away

Reply:
Done! c: